VideoLocker/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -228,6 +233,18 @@ public String getNewRelicKey() {
     }
 
     /**
+     * End to end test configurations
+     */
+    public class EndToEndConfig {
+        private String DEFAULT_EMAIL_TEMPLATE = "test-{unique_id}@example.com";
```

Comment:
This should be a `static` class (along with all the other model classes defined here 😦), and the template should be a `static final` constant.